server/repository/migrations/postgres/2022-01-27T08-00_create_changelog_table/up.sql

```diff
@@ -10,6 +10,7 @@ CREATE TYPE changelog_table_name AS ENUM (
     'stocktake_line',
     'requisition',
     'requisition_line'
+    'activity_log'
```

Comment:
for those of us who are re-using databases and wanting to test.. `ALTER TYPE changelog_table_name ADD VALUE IF NOT EXISTS 'activity_log';`

Reply:
I've also noticed that the `up` for `activity_log` has the column `type log_type` but the enum which is created above that is `activity_log_type`. Can you check and see which one it should be please?

Reply:
Opps...fixed 🙏